opentelemetry/src/logs/record.rs

```diff
@@ -59,11 +59,11 @@ pub enum AnyValue {
     /// A boolean value
     Boolean(bool),
     /// A byte array
-    Bytes(Vec<u8>),
+    Bytes(Box<Vec<u8>>),
```

Comment:
if feasible, could we make this in a separate, small PR.
or update the changelog to indicate this change.